esp_delta_ota/README.md

```diff
@@ -0,0 +1,31 @@
+# ESP Delta OTA
```

Comment:
We should highlight some of the advantages of the design here:

1. No additional storage partition required for the "patch" file
2. Only firmware level changes required for integrating this component, no bootloader changes required
3. This scheme can be implemented for the existing devices in the field
4. Minimal memory overhead (please mention number) with this component